### ISSUE TITLE
Stable version issue with new custom Sector

### ISSUE THREAD

**Issue by flyteach** [Grauenwolf/TravellerTools#114]
Hello again,
A new sector and I'm hoping you can fix it. This is such an awesome tool! 
My new sector (Sagittarius Cluster) comes in fine to the v2 demo version, but is not showing in the list for the stable version. Is there a way to get that fixed?
Again, thanks for this and hoping the work continues!

**Comment by Grauenwolf** [comment]
I might just need to restart the server so it downloads a fresh copy of the sector list.

Go ahead and try it again. If it doesn't work this time, send me a link to it from the v2 version.

**Comment by flyteach** [comment]
Hello Jonathan,

Thanks for your help. It is still not showing in the list for v1. Here's a link to a world in the Sagittarius cluster from v2.

https://travellertoolsdemo.azurewebsites.net/world/M1105/-146,-20/0210/info

Not sure how I can send one for the whole sector. Oh, wait, tried the "sector info" and here it is:

https://travellertoolsdemo.azurewebsites.net/world/M1105/-146,-20

FWIW, here's a link to it in Travellermap:

https://travellermap.com/?p=-4032.839!819.326!7.3&options=58359&mains=1

Flyteach/Jim

**Comment by Grauenwolf** [comment]
![image](https://github.com/Grauenwolf/TravellerTools/assets/10627661/acd43135-297a-4d40-aa5b-b9578cb76f71)

No subsector data found?

How is this working at all? You shouldn't be able to get to it with either version.

**Comment by Grauenwolf** [comment]
So there really are no subsectors coming back from TravellerMap.

![image](https://github.com/Grauenwolf/TravellerTools/assets/10627661/9d52e335-884f-4414-9b79-8fc737df5b5a)

**Comment by Grauenwolf** [comment]
Ok, so what I'm going to do is invent fake sector names "A" thru "P" to account for the missing data.

Easy fix. 

Now before you run off to play with your new toy, please explain to me what "My new sector" means. Like, can you just create new sectors in Traveller Map?

**Comment by flyteach** [comment]
Hello Jonathan,
Thank you so much. I did not realize I was missing the subsection info as it wasn't necessary for Traveller Map. In this area, the subsectors are not as important as the political and cultural boundaries, so I skipped it. Now I know I should at least create that metadata with just the subsector letters for names probably to fill out the info. Sorry for the confusion, rookie mistake, very much appreciate your help and understanding.
"My new sector" is exactly that. Joshua allows GMs (or anyone else) to create sectors that can be uploaded to TravellerMap. They are not official and are placed far away from the OTU, but perfectly viable and workable areas that can be leveraged by GMs and players. Specifically he has a note about it here: https://travellermap.com/doc/submit.
As a VERY old school player (40+ years) I have more invested in the background I started creating in college and can now leverage the online tools I dreamed of back then. Your Traveller Tools are an absolutely FANTASTIC addition to making the life of a busy GM more effective and more fun. Again, thank you for all your hard work.
Jim/Flyteach
